### PR TITLE
feat(@angular-ru/eslint-plugin): add custom eslint rules for projects

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/eslint-plugin/src/interfaces/eslint-rule.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Rule } from 'eslint';
+
+export interface EslintRule {
```
Comment (splincode) [inline packages/eslint-plugin/src/interfaces/eslint-rule.ts:3]: EslintRules

File: packages/eslint-plugin/src/rules/no-model-suffix-rule.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Any } from '@angular-ru/common/typings';
+import { Rule } from 'eslint';
+
+import { fileEndsWith } from '../utils/file-ends-with';
+
+export function noModelSuffixRule(): Any {
```
Comment (splincode) [inline packages/eslint-plugin/src/rules/no-model-suffix-rule.ts:6]: don't any

File: packages/eslint-plugin/src/utils/file-ends-with.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export function fileEndsWith(context: import('eslint').Rule.RuleContext, suffix: string): boolean {
```
Comment (splincode) [inline packages/eslint-plugin/src/utils/file-ends-with.ts:1]: `import('eslint').Rule.RuleContext` ...

**Review by evigkein — APPROVED** [review]

**Review by evigkein — DISMISSED** [review]

File: packages/eslint-config/index.js [hunk 4]
```diff
@@ -527,7 +534,7 @@ module.exports = {
                 // 'sonarjs/prefer-default-last': 'error',
                 // 'sonarjs/prefer-promise-shorthand': 'error',
                 // 'sonarjs/prefer-type-guard': 'error',
-                // 'sonarjs/use-type-alias': 'error'
+                // 'sonarjs/use-type-alias': 'error',
```
Comment (Schirbak) [inline packages/eslint-config/index.js:537]: is this file ignored by prettier?

File: packages/eslint-plugin/src/rules/no-model-suffix-rule.ts (hunk 2 above)
Comment (Schirbak) [reply packages/eslint-plugin/src/rules/no-model-suffix-rule.ts:7]: bump

File: packages/eslint-plugin/src/interfaces/eslint-rule.ts (hunk 1 above)
Comment (Schirbak) [reply packages/eslint-plugin/src/interfaces/eslint-rule.ts:3]: bump

File: packages/eslint-plugin/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+$ npm install @angular-ru/eslint-plugin -D
+```
+
+Add to your `.eslintrc.json`:
+
+```json
+{
+    "plugins": ["@angular-ru/eslint-plugin"],
+    "rules": {
+        "@angular-ru/no-model-suffix": "error",
```
Comment (splincode) [inline packages/eslint-plugin/README.md:27]: need to fix

```ts
"@angular-ru/no-suffix-file": [ "error", "*.model.ts", "*.enum.ts" ],
```

File: packages/eslint-config/index.js (hunk 4 above)
Comment (splincode) [reply packages/eslint-config/index.js:536]: we shouldn't ignore this file

**Review by nikitasadchikov — APPROVED** [review]

**Pull request merged**